### DISCUSSION TITLE
Dominican Republic flag has 2 different shades of red when there should be one

### DISCUSSION THREAD

**Discussion by Skillosu** [ppy/osu#14626] [Ideas] [answered]
Pretty self explanatory, https://osu.ppy.sh/assets/images/flags/1f1e9-1f1f4.svg the flag has 2 different shades of red when there should be one as you can see here:
![image](https://user-images.githubusercontent.com/55836487/132093693-568725b2-478f-463c-a250-bd0c6f941a6c.png)

I dont use github so i didnt know where else to post this, sorry if i put this in the wrong category.

**Comment by bdach** [comment, marked as answer]
The icons are from https://github.com/twitter/twemoji. Take it up with them.
